content/developer/reference/cmdline.rst

```diff
@@ -272,6 +277,14 @@ Emails
 
     Password to connect to the SMTP server
 
+.. option:: --smtp-ssl-certificate-filename <SSL certificate>
+
+    SSL certificate to use for the authentication. If set, "smtp-ssl-private-key" is required.
+
+.. option:: --smtp-ssl-private-key-filename <SSL private key>
+
+    SSL private key to use for the authentication. If set, "smtp-ssl-certificate" is required.
```

Comment:
Let's change it to: 

` SSL private key is to be used for authentication. If set, then "smtp-ssl-certificate" is required.'

The reason I made this small change is to make the sentence flow a little bit better while keeping the context.